docs/variables.md

```diff
@@ -113,3 +113,36 @@ variable via `resources.inputs.<resourceName>.<variableName>` or
 | `name` | The name of the resource. |
 | `type` | Type value of `"cloudEvent"`. |
 | `target-uri` | The URI to hit with cloud event payloads. |
+
+## Variables that can be substituted in a CRD
+
+| CRD | Field |
+| --- | ----- |
+| `Task` | `credentials.path` | 
+| `Task` | `spec.steps[].name` |
+| `Task` | `spec.steps[].image` |
+| `Task` | `spec.steps[].env.value` |
+| `Task` | `spec.steps[].env.valuefrom.secretkeyref.name` |
+| `Task` | `spec.steps[].env.valuefrom.secretkeyref.value` |
+| `Task` | `spec.steps[].env.valuefrom.configmapkeyref.name` |
+| `Task` | `spec.steps[].env.valuefrom.configmapkeyref.value` |
+| `Task` | `spec.steps[].volumemounts.name` |
+| `Task` | `spec.steps[].volumemounts.mountpath` |
+| `Task` | `spec.steps[].volumemounts.submountpath` |
+| `Task` | `spec.volumes[].name` |
+| `Task` | `spec.volumes[].configmap.name` |
+| `Task` | `spec.volumes[].secret.secretname` |
+| `Task` | `spec.volumes[].persistentvolumeclaim.claimname` |
+| `Task` | `spec.volumes[].projected.sources.configmap.name` |
+| `Task` | `spec.volumes[].projected.sources.secret.name` |
+| `Task` | `spec.volumes[].projected.sources.serviceaccounttoken.audience` |
```

Comment:
These were also very recently added:

```
spec.volumes[].csi.nodepublishsecretref.name
spec.volumes[].csi.volumeattributes.*
```

That second one is a map of unspecified string keys so I'm not sure what the best way is to express it other than to use an `*`.